rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -1960,4 +1961,33 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
       Vector(MethodNotDefined("add", "Map"))
     )
   }
+
+  "Running out of phlogistons" should "stop the evaluation" in {
+    implicit val errorLog = new ErrorLog()
+
+    val test = withTestSpace(errorLog) {
+      case TestFixture(space, reducer) =>
+        implicit val env = Env.makeEnv[Par]()
+        reducer.setAvailablePhlos(Cost(5)).runSyncUnsafe(1.second)
+        val splitRand = rand.splitByte(0)
+        val receive =
+          Receive(
+            Seq(
+              ReceiveBind(
+                Seq(ChanVar(FreeVar(0)), ChanVar(FreeVar(1)), ChanVar(FreeVar(2))),
+                Quote(GString("channel"))
+              )
+            ),
+            Par(),
+            false,
+            3,
```

Comment:
What does 3 represent?

Reply:
Number of free variables in the expression.

Reply:
ok. nevermind